cmd/sachet/config.go

```diff
@@ -24,6 +25,7 @@ type ReceiverConf struct {
 	Provider string
 	To       []string
 	From     string
+	Text     string
```

Comment:
What do you think about renaming this to `Template`?

Reply:
@marcelcorso Agreed, thanks for clarifying!

Reply:
I made everything to be similar to alertmanager configuration and thought about some  providers that support message titles or something like that.

I'd prefer current name but don't mind renaming.

Reply:
@r3nic1e No, this is perfect - I wasn't aware that's what Prometheus calls it, so let's be consistent then 👍